content/en/apps/guides/data/analytics/setup.md

```diff
@@ -0,0 +1,108 @@
+| `CHT_PIPELINE_BRANCH_URL` | `"https://github.com/medic/cht-pipeline.git#main"` |                                                                                                                                                |
+| `COUCHDB_USER`            | `medic`                                            | Username of the CouchDB instance to sync with                                                                                                  |
+| `COUCHDB_PASSWORD`        | `password`                                         | Password of the CouchDB instance to sync with                                                                                                  |
+| `COUCHDB_DBS`             | `"medic"`                                          | Space separated list of databases to sync e.g `"medic medic_sentinel"`                                                                         |
+| `COUCHDB_HOST`            | `couchdb`                                          | Host of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_PORT`            | `5984`                                             | Port of the CouchDB instance to sync with                                                                                                      |
+| `COUCHDB_SECURE`          | `false`                                            | Is connection to CouchDB instance secure?                                                                                                      |
+
+{{% alert title="Note" %}}
+If `CHT_PIPELINE_BRANCH_URL` is pointing to a private repo then you need to provide an access token in the url i.e. `https://<PAT>@github.com/medic/cht-pipeline.git#main`. In this example you will replace `<PAT>`  with an access token from Github. Instruction on how to generate one can be found [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+{{% /alert %}}
+
+### Setup
+The following configurations facilitate data synchronization, transformation, and storage for local development and testing.
+
+Copy the values in `env.template` file to the `.env` file and update them accordingly to the local configuration for the different scenarios below.
+
+Install the dependencies:
+```sh
+npm install
+```
+
+#### With sample CouchDB data
+This setup involves starting Logstash, PostgreSQL, PostgREST, DBT, and CouchDB. Sample fake data is generated for CouchDB.
+
+Run the Docker containers and wait for every container to be up and running:
+```sh
+# starts logstash, postgres, postgrest, generator (for fake data), couchdb and dbt
+npm run local
+```
+
+This is equivalent to running:
+```sh
+# starts logstash, postgres, postgrest, generator (for fake data), couchdb and dbt
+docker-compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f docker-compose.yml up -d
+```
```

Comment:
I think this is too much information and might be confusing to someone who wonders if they should run this, as we're presenting it in the "copy an paste this code" format.

Instead, recommend a deep link:

```suggestion
This is a wrapper for some `docker compose` calls.  To find out exactly what is being called, see [the source script](https://github.com/medic/cht-sync/blob/main/scripts/config.ts#L40). 
```

Reply:
I actually removed the `npm run local` instruction and kept only the docker compose calls, for the sake of consistency. @witash is working on removing the node script.